airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -73,10 +73,11 @@ def __init__(
         secret_arn: str | None = None,
         statement_name: str | None = None,
         with_event: bool = False,
-        await_result: bool = True,
+        await_result: bool | None = None,
         poll_interval: int = 10,
         aws_conn_id: str = "aws_default",
         region: str | None = None,
+        wait_for_completion: bool = True,
```

Comment:
Small nit, could we move in `await_result` in the end of operator arguments? And in this place we could store `wait_for_completion`

This not affect to users because it is mandatory requirement for all operators to send arguments as key-value arguments and I think it would be nice to store all deprecated values in the bottom operator constructor definition.